### PR TITLE
Catalog Maintenance

### PR DESCRIPTION
**Pull request by calebhailey**
- [x] fix system/disk-monitoring integration 
  - [x] fix `resource_patches` (should be an array)
  - [x] delete current tag
- [x] update integration prompts `required` field (tl;dr: it should be `prompt.required`, not `prompt.input.required`)
- [x] update the README to disambiguate variables vs templates (`[[ ]]`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,35 +262,36 @@ spec:
...
+    * **`input.title`** (required): input field title/label, displayed above the input field.
+    * **`input.description`** (optional): input field description, displayed below the input field.
+    * **`input.format`** (optional): input value display format; allowed values: `sh`, `ecmascript-5.1`, `cron`, `duration`, `tel`, `email`, `url`, `hostname`, `ipv4`, `ipv6`, `envvar`, `sha-256`, `sha-512`, `io.sensu.selector`. Some display formats provide helpers to simplify user input.
+    * **`input.ref`** (optional): Sensu API resource reference in `<api_group>/<api_resource>/<api_field_path>` format. For example, `core/v2/Pipeline/metadata/name` refers to `core/v2` API group `Pipeline` resources, which will be presented to the user in a drop-down selector; once selected, the value of the `metadata/name` field will be captured as the input value.
+    * **`input.filter`** (coming soon): Sensu API resource reference filters in [Sensu Query Expression (SQE)] format; e.g. `.labels.provider == "alerts"`. Used to filter the results of a `ref`.
```
Comment (jamesdphillips) [inline README.md:294]: Let's go with `refFilter`

**Review by jamesdphillips — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (calebhailey) [reply README.md:294]: @jamesdphillips updated the readme and a few integrations that have this property set in their prompts; can you look at that commit and let me know if the refFilter values look right (so you'll have something to test)?

Comment (jamesdphillips) [reply README.md:294]: appears to be correct!

Comment (calebhailey) [reply README.md]: awesome; thx! 🤞

**Pull request merged**